rhemyn/compile.js

```diff
@@ -189,8 +203,19 @@ const generateChar = (node, negated, get, stringSize) => {
   }
 
   return [
+	// if len - counter <= 0, break
+	[ Opcodes.local_get, Length ],
+	[ Opcodes.local_get, Counter ],
+	[ Opcodes.i32_sub ],
+	...number(0, Valtype.i32),
+	[ Opcodes.i32_gt_s ],
+	...number(1, Valtype.i32),
+	[ Opcodes.i32_xor ],
```

Comment:
Why `((... > 0) ^ 1)`? There's a `i32_le_s` op for `<=`?

Reply:
woops that's my bad 😅